packages/navie/src/explain.ts

```diff
@@ -112,8 +112,8 @@ export interface ClientRequest {
 }
 
 export class ExplainOptions {
-  modelName = 'gpt-4-0125-preview';
-  tokenLimit = 8000;
+  modelName = process.env.NAVIE_MODEL ?? 'gpt-4-0125-preview';
```

Comment:
If we expose configuration via env var, I feel that we should prefix all our variables with `APPMAP_`.